terraform/modules/aws/gateway/main.tf

```diff
@@ -0,0 +1,52 @@
+locals {
+  application_name    = var.application_name != null ? var.application_name : var.image
+  application_version = var.application_version != null ? var.application_version : var.image_tag
+
+  environment_variables = concat([
+    {
+      name  = "RUST_LOG"
+      value = var.observability_log_level
+    },
+    {
+      name  = "RUST_BACKTRACE"
+      value = "full"
+    },
+    {
+      name  = "FIREZONE_TOKEN"
+      value = var.token
+    },
+    {
+      name  = "FIREZONE_API_URL"
+      value = var.api_url
+    },
+    {
+      name  = "FIREZONE_ENABLE_MASQUERADE"
+      value = "1"
+    }
+  ], var.application_environment_variables)
+}
+
+resource "aws_instance" "this" {
+  ami                         = var.ami
+  instance_type               = var.instance_type
+  monitoring                  = var.monitoring
+  subnet_id                   = var.subnet_id
+  vpc_security_group_ids      = var.vpc_security_group_ids
+  associate_public_ip_address = var.associate_public_ip_address
+  private_ip                  = var.private_ip
+  key_name                    = var.key_name
+
+  #user_data = file("${path.module}/scripts/setup.sh")
```

Comment:
accidental cruft?

Reply:
Ah, yep, good catch!  Was flipping back and forth between a cloud-init.yaml and a shell script.  Will get this removed.